test/integration/sugar-mixins.spec.js

```diff
@@ -11,13 +12,11 @@ describe('Sugar mixins', () => {
     config.mixins = [{
       types: 'Global',
       extend: {
-        createAppAndLoad(appName, scriptName) {
+        createAppAndLoad(appName, loadScript) {
           return qixGlobal.createApp(appName).then(appInfo =>
```

Comment:
Shouldn't we switch to using session apps, or you mean to do that in another PR?